.env

```diff
@@ -8,6 +8,8 @@ ESIGNET_HOST=https://api.qa-inji.mosip.net
 
 GOOGLE_NEARBY_MESSAGES_API_KEY=
 
+OBSRV_HOST = https://dataset-api.obsrv.mosip.net
+
```

Comment:
It has to be added in https://github.com/mosip/inji/blob/develop/types/react-native-dotenv/index.d.ts